tests/integrationv2/configuration.py

```diff
@@ -34,7 +34,6 @@
     Curves.P384,
 ]
 
-
```

Comment:
Double blank lines is actually correct in python! :) https://www.python.org/dev/peps/pep-0008/#blank-lines